scripts/autogen

```diff
@@ -26,6 +26,21 @@ montgomery_factor = pow(2, 16, modulus)
 # - #undef's for CU-local macros
 
 
+def set_cwd_to_git_root():
+    current_path = os.getcwd()
+    git_root = None
+    while current_path != "/":
+        if os.path.exists(os.path.join(current_path, ".git")):
+            git_root = current_path
+            break
+        current_path = os.path.dirname(current_path)
+
+    if git_root is None:
+        raise Exception("Could not find root of repository")
+
+    os.chdir(git_root)
```

Comment:
Why not just 
```
os.chdir(os.path.join(os.path.dirname(__file__), ".."))
```
? 
feels much cleaner to me.

Reply:
Fair point!

Reply:
Done